x-pack/plugins/monitoring/public/components/logs/logs.js

```diff
@@ -190,8 +194,9 @@ export class LogsContent extends PureComponent {
     if (!enabled || !show) {
       return null;
     }
+    const discoverLink = getDiscoverLink(clusterUuid, nodeId, indexUuid, sharePlugin, logsIndices);
 
-    return infra ? (
```

Comment:
ℹ️ Rendering the callout was guarded by whether `infra` is available or not, which doesn't make a lot of sense anymore since we provide a link to discover, not infra here.

So it's now guarded by whether we can successfully generate a discover link or not.